tools/src/main/java/it/eng/tools/rest/api/ApplicationPropertiesAPIController.java

```diff
@@ -0,0 +1,89 @@
+package it.eng.tools.rest.api;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import it.eng.tools.exception.ApplicationPropertyNotChangedAPIException;
+import it.eng.tools.exception.ApplicationPropertyNotFoundAPIException;
+import it.eng.tools.model.ApplicationProperty;
+import it.eng.tools.model.Serializer;
+import it.eng.tools.service.ApplicationPropertiesService;
+import lombok.extern.java.Log;
+
+@RestController
+@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, path = "/api/connector_property")

```

Comment:
@scs-eng You had some suggestion about API's, how url should look like. Any suggestion for this mapping, connector_property
Should we change others to be like this, with _ or to change this to camel case?

Reply:
This one is still relevant
Other endpoints does not include '_' in address

Reply:
As REST API best practise, it's suggested to use hyphens for a better readability. It is preferable to underscore and camel case.